lua/neogit/popups/rebase/actions.lua

```diff
@@ -106,6 +106,22 @@ function M.interactively(popup)
   end
 end
 
+function M.reword(popup)
+  local commit
+  if popup.state.env.commit then
+    commit = popup.state.env.commit
+  else
+    commit = CommitSelectViewBuffer.new(git.log.list()):open_async()[1]
+    if not commit then
+      return
+    end
+  end
+  local old_message = git.log.message(commit)
+  local new_message = vim.fn.input("Message: ", old_message)
```

Comment:
Line 2 in this file we import the input lib - use that for getting input.

Reply:
Done